clmm/theory/ccl.py

```diff
@@ -55,23 +56,24 @@ def __init__(self, massdef='mean', delta_mdef=200, halo_profile_model='nfw',
             'mean': 'matter',
             'critical': 'critical',
             'virial': 'critical'}
-        self.hdpm_dict = {'nfw': ccl.halos.HaloProfileNFW}
+        self.hdpm_dict = {'nfw': ccl.halos.HaloProfileNFW,
+                          'einasto': ccl.halos.HaloProfileEinasto,
+                          'hernquist': ccl.halos.HaloProfileHernquist}
         self.cosmo_class = CCLCosmology
-        # Uncomment lines below when CCL einasto and hernquist profiles are stable
-        # (also add version number)
-        # if version.parse(ccl.__version__) >= version.parse('???'):
-        #    self.hdpm_dict.update({
+        # Uncomment lines below when CCL einasto and hernquist profiles are stable (also add version number)
+        #if version.parse(ccl.__version__) >= version.parse('???'):
```

Comment:
Are we always going to require am installed CCL version with Einasto and Hernquist profiles? It this why is this line still commented?

Reply:
@combet @m-aguena For your information, the halos modules were added to CCL in version 2.0.2.dev84. An official release with Einasto and Hernquist profiles would be 2.1.0.

Reply:
Currently pip-installable ccl release is 2.3.0, so I think it's OK to request >=2.1.0